beniget/beniget.py

```diff
@@ -1025,11 +1025,19 @@ def visit_UnaryOp(self, node):
     def visit_Lambda(self, node, step=DeclarationStep):
         if step is DeclarationStep:
             dnode = self.chains.setdefault(node, Def(node))
+            for default in node.args.defaults:
```

Comment:
While reading this I realize that we also don't visit argument annotations for lambda in case of `future_annotations`.

What do you think of aggregating the behavior of lambda and functiondef analysis in a helper function?

Reply:
Well it would have avoided this patch ;-)